ui/src/apps/clusterInfo/components/ClusterInfoTable.js

```diff
@@ -101,6 +102,7 @@ function wrapNode(node, comp, id, t) {
     )
   }
   return {
+    key: `${comp}:${node.ip}:${node.port}`,
```

Comment:
i guess ip+port is enough for a key.

Reply:
no, I'll tried it, but different components may have same key, and it will raise warning.

Reply:
why will different compoennt have same ip+port?

Reply:
Well, thats my testing config...I think leave a comp here doesn't increase any complexity, and it's proper because it's just a sub-key of `comp`.